guard/README.md

```diff
@@ -281,7 +281,7 @@ Autocompletions are only something available for version >= 3.0
 #### zsh
 
 ```sh
-    cfn-guard completions --shell='zsh' > usr/local/share/zsh/site-functions/_cfn-guard && compinnit
```

Comment:
One more change: we should change `compinnit` to `compinit` for it to work

Reply:
done